### ISSUE TITLE
MyY throws fatal error after installation

### ISSUE THREAD

**Issue by alexschedrov** [ymcatwincities/openy#1920]
When I go to the fresh Open Y installation/Sandbox and try to enable "OpenY MyY" it throws fatal error after installation.

Also would be nice to have documentation/demo on how to use the MyY feature.

**Comment by paige-kiecker** [comment]
MyY is in alpha state and
- we need to know which error it is throwing ( probably memory limit, which is acceptable as for alpha )
- this will be fixed in a scope of adopting MyY for YGTC

**Comment by alexschedrov** [comment]
@paige-kiecker sounds good! 👍

The error is pretty generic:

![image](https://user-images.githubusercontent.com/1316234/74854224-b6a0db00-5347-11ea-852f-f6dc3f16a750.png)

**Comment by sarah-halby** [comment]
@ymcagtc our recommendation for this module is to decouple from the main repository due to low demand from customers and lack of demo and documentation from Y North. Once My Y is ready and it is approved by the Product Committee, we can move this item out of the backlog. What are your thoughts? cc @Sanchiz & @paige-kiecker @podarok
